fai.ps1

```
@@ -319,7 +319,11 @@ function Compose-Restart([string]$svc = "") {
 }
 
 function Compose-ExecAi([string[]]$cmdArgs) {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Compose-ExecAi' uses an unapproved verb.

(PSUseApprovedVerbs)